backend/util/tracer.go

```diff
@@ -1,103 +1,128 @@
 package util
 
-// This schema/arch is taken from: https://github.com/99designs/gqlgen/blob/master/graphql/handler/apollotracing/tracer.go
-
 import (
 	"context"
-	"encoding/json"
-	"time"
+	"fmt"
 
-	"github.com/99designs/gqlgen/graphql"
-	log "github.com/sirupsen/logrus"
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"github.com/highlight/highlight/sdk/highlight-go"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
-type Tracer struct {
-	graphql.HandlerExtension
-	graphql.ResponseInterceptor
-	graphql.FieldInterceptor
-
-	serverType Runtime
+type MultiSpan struct {
+	ddSpan tracer.Span
+	hSpan  trace.Span
 }
 
-func NewTracer(backend Runtime) Tracer {
-	return Tracer{serverType: backend}
+func (s *MultiSpan) Finish(err ...error) {
+	// TODO: Clean this up
```

Comment:
anything else here? maybe if there is more than one error passed we can use the first one but `log` a warning?

Reply:
Just added some logging in https://github.com/highlight/highlight/pull/6616/commits/1bd8daf39a9eabb469b5a345450bd3c0e6f79378.